### PR TITLE
Use nested name fields in Add/Edit form

### PR DESCRIPTION
**Pull request by rvsia**
**Description**

Gets UI ready for https://github.com/ManageIQ/sources-api/pull/74

**TODO**
- [x] Add (after https://github.com/RedHatInsights/frontend-components/pull/132 )
- [X] Edit
- [x] Wating for https://github.com/data-driven-forms/react-forms/pull/102 (tests will fail until it)

### CODE REVIEW

**Comment by rvsia** [comment]
@Hyperkid123 Please review, but merge it after https://github.com/ManageIQ/sources-api/pull/74

- Only Openshift and AWS are using new fields name now, summary step won't work properly until the API PR is merged

File: src/api/entities.js [hunk 1]
```diff
@@ -94,13 +94,9 @@ const urlOrHost = formData => formData.url ? parseUrl(formData.url) : formData;
 export function doUpdateSource(source, formData) {
     const inst = getSourcesApi();
 
-    let sourceData = {
-        name: formData.source_name
-    };
-
-    return inst.updateSource(source.id, sourceData)
+    return inst.updateSource(source.id, formData.source)
```
Comment (Hyperkid123) [inline src/api/entities.js]: one line

**Comment by Hyperkid123** [comment]
Looks good, but I would like to test it with the required PR.

**Comment by gtanzillo** [comment]
@Hyperkid123 @rvsia I'm ready to merge https://github.com/ManageIQ/sources-api/pull/74. Perhaps we can merge that PR and this one tomorrow while you're both available to test and address and issues that may come up.

**Comment by rvsia** [comment]
@gtanzillo I agree! 👍

**Pull request merged**